### ISSUE TITLE
Classes do not support CBA_Extended_EventHandlers

### ISSUE THREAD

**Issue by MrDell61** [CBATeam/CBA_A3#955]
**Arma 3 Version:** `1.82.144872` (stable profiling)
**CBA Version:** `3.7.1` (stable)

**Mods:**
```
- CBA_A3
```

**Description:**
- Classes do not support CBA_Extended_EventHandlers (https://pastebin.com/MFQyBYYu)
I could not find the information that they were not supported. Also, I did not find deleting the eventHandlers class in config.cpp.
Is it worth worrying if the developer of another mod, also accidentally removes this class?

**Steps to reproduce:**
- `_noSupportClass = [];
{
   if !(isClass (_x >> "EventHandlers" >> "CBA_Extended_EventHandlers")) then {_noSupportClass pushBack _x};
} forEach ("true" configClasses (configFile >> "cfgVehicles"));
copyToClipboard str _noSupportClass`
`
class Extended_InitPost_EventHandlers
{
	class Flag_IDAP_F
	{
		// Make sure everyone is happy when the mission starts
		class test
		{
			init = "diag_log 'Flag_IDAP_F init'";
		};
	};
};
`

**Where did the issue occur?**
- Dedicated / Self-Hosted Multiplayer / Singleplayer / Editor (Singleplayer) / Editor (Multiplayer) / Virtual Arsenal

**RPT log file:**
- https://pastebin.com/Wz9v9Jxg

**Comment by commy2** [comment]
What?

**Comment by MrDell61** [comment]
Xeh eventhandlers does not work for these classes https://pastebin.com/MFQyBYYu

**Comment by commy2** [comment]
That's true. It is disabled on buildings for performance reasons. See: https://github.com/CBATeam/CBA_A3/issues/311#issuecomment-212285477

Close as duplicate?
